tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -37,6 +38,15 @@ def get_next_input(self):
     def get_next_batch_size(self):
         return self.batch_sizes[self.current_group_id]
 
+def import_cv2():
+    global cv2
```

Comment:
I'm not sure that it is required to use it globally, because all opencv calls localized in one function, it will be enough to have it in inner scope only